src/status_im/chat/models/mentions.cljs

```diff
@@ -173,8 +173,10 @@
 (defn get-mentionable-users
   [{{:keys          [current-chat-id]
      :contacts/keys [contacts] :as db} :db}]
-  (let [{:keys [chat-type users] :as chat}
+  (let [{:keys [chat-type community-id users] :as chat}
         (get-in db [:chats current-chat-id])
+        mentionable-commmunity-members @(re-frame/subscribe
```

Comment:
Is that an appropriate way to use subs? I know I did this once with mentions as well, but as I remember @flexsurfer was strongly against it.

Reply:
Hi @rasom,
Yes you are right, I can create a function that will give mentionable members list.
And I can use that function here and also in subs.
Is that you meant?
Thanks

Reply:
@Parveshdhull , yes, something like this